pkgs/by-name/fl/flye/package.nix

```diff
@@ -0,0 +1,47 @@
+{ lib
+, fetchFromGitHub
+, python3Packages
+, zlib
+, curl
+, libdeflate
+, bash
+, coreutils
+}:
+
+python3Packages.buildPythonApplication rec {
+  pname = "flye";
+  version = "2.9.3";
+
+  src = fetchFromGitHub {
+    owner = "fenderglass";
+    repo = pname;
```

Comment:
usage of pname is considered antipattern https://github.com/NixOS/nixpkgs/issues/277994